apps/timetrace/ttcat

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/python3
```

Comment:
no .py extension?

Reply:
Since it's a script one might want in their path, I think people would rather type `ttcat` than `ttcat.py`.  And who knows?  We may soon convert it to some language other than Python.